charts/gateway/templates/deployment.yaml

```diff
@@ -477,6 +477,14 @@ spec:
             - key: hazelcast-xml
               path: hazelcast-client.xml
 {{- end }}
+{{ if .Values.disklessConfig  }}
+        - name: { { template "gateway.fullname" . } }-node-properties
+          secret:
+            secretName: { { template "nodePropertiesSecretName" . } }
+            items:
+              - key: node.properties
+                path: node.properties
+{{ end }}
```

Comment:
Actual volume mount is missing. License example: https://github.com/CAAPIM/apim-charts/blob/node-properties/charts/gateway/templates/deployment.yaml#L189

Reply:
Updated deployment.yaml